### ISSUE TITLE
Directory Structure needs to be modified.

### ISSUE THREAD

**Issue by sidntrivedi** [denolib/guide#8]
The doc mentions various directories that have been modified in the current deno codebase. For example : `src/, build_extra/`. PTAL. Thanks

**Comment by sidntrivedi** [comment]
I will be really glad to help out to modify the doc, but being a newbie would need some help. It has been only few days since I have started looking into the deno codebase. Thanks.

**Comment by kevinkassimo** [comment]
Yeah this doc is pretty outdated. Do you want to collaborate and rewrite some of the chapters? I plan to spend some time working on this December (as recent months there are still quite a few pending major internal refactoring)

**Comment by kevinkassimo** [comment]
Since we need a complete rewrite, I am temporarily holding your PRs. If you want to collaborate, probably take some reading the codebase and ask others in the Deno Gitter chatroom (I'm also there).

I am personally quite busy till mid December, after which we can try to update this together if you want.

**Comment by sidntrivedi** [comment]
> Yeah this doc is pretty outdated. Do you want to collaborate and rewrite some of the chapters? I plan to spend some time working on this December (as recent months there are still quite a few pending major internal refactoring)

Sure, I would love to collaborate and help rewrite the doc. It will surely help me learn a lot. 😃

**Comment by sidntrivedi** [comment]
> Since we need a complete rewrite, I am temporarily holding your PRs. If you want to collaborate, probably take some reading the codebase and ask others in the Deno Gitter chatroom (I'm also there).
> 
> I am personally quite busy till mid December, after which we can try to update this together if you want.

Okay, will firstly understand the project and try to contribute myself and then will join you in updating it in December. Thanks :)
